assets/app/protected/users/teams/index/route.js

```diff
@@ -34,5 +34,22 @@ export default Ember.Route.extend({
       let teamController = this.controllerFor('protected.users.teams');
       teamController.send('refreshTeamsData');
     }
+  },
+
+  model() {
+    return this.get('store').query('file', { filename: './', teams: true })
+      .catch((err) => {
+        if (err.errors.length === 1 && err.errors[0].code === '000007') {
```

Comment:
This was error checking from the old backend and is not relevant here. You safely delete it a replace it with a toast.